SwiftUIBasics/ContentView.swift

```diff
@@ -17,5 +17,5 @@ struct ContentView: View {
 }
 
 #Preview {
-    ContentView()
+    SignUpView()
```

Comment:
You dont need to run SignUpView() on the content view to test it, you can always test on the preview window